windows-release/msi-steps.yml

```
@@ -114,3 +114,26 @@ steps:
   - publish: '$(Build.ArtifactStagingDirectory)\msi'
     artifact: msi
     displayName: 'Publish MSI'
+
+  - powershell: >
+      $(Python)
```

```suggestion
      & $(Python)
```

I'm 99% sure you'll need this. The Python variable here has been exported by an earlier step that gets it from one of our scripts that assumes it's running in `cmd.exe`, so it's possible that the variable contains a quoted path. In Powershell, this looks like a string literal, so it won't treat it as a quoted command unless you add the `&` (which, IIUC, just means "treat this entire input as a command").

---

Done in https://github.com/python/release-tools/pull/100/commits/c01765dacd7876de68e712362e2f8015aafae02b, thanks for the Powershell knowledge!